app/src/main/java/zapsolutions/zap/util/RemoteConnectUtil.java

```diff
@@ -97,22 +97,29 @@ public static void saveRemoteConfiguration(RemoteConfiguration config, @Nullable
         try {
             if (config instanceof LndConnectConfig) {
                 LndConnectConfig lndConfig = (LndConnectConfig) config;
+
+                int port = lndConfig.getPort();
+                if (port == 8080) {
```

Comment:
I'm not sure if we should just switch ports by ourselves without asking the user first. 
Maybe some users have set up this port by accident (e.g. when setting up a tor hidden service), and then cannot connect if we just rewrite it in the code.
@michaelWuensch